ratis-client/src/main/java/org/apache/ratis/client/impl/RaftClientImpl.java

```diff
@@ -105,7 +105,12 @@ public int getAttemptCount() {
     }
 
     public ClientRetryEvent newClientRetryEvent(RaftClientRequest request, Throwable throwable) {
+      return newClientRetryEvent(request, throwable, false);
+    }
+
+    ClientRetryEvent newClientRetryEvent(RaftClientRequest request, Throwable throwable, boolean get) {
```

Comment:
Is there a plan to call this with `get=true` somewhere?  I didn't see it in current code.

Reply:
You are right -- `get` is always false after refactoring.  Let me combine the `newClientRetryEvent` methods.